docs/cicd.md

```diff
@@ -14,40 +14,40 @@
 
 The easiest way to deliver a new build to production or development is to trigger the corresponding CircleCI workflows *triggered_native_development_delivery* and *triggered_production_delivery*:
 
-* Get a CircleCI [Personal API Token](https://circleci.com/docs/2.0/managing-pi-tokens/).
+* Get a CircleCI [Personal API Token](https://circleci.com/docs/2.0/managing-api-tokens/).
 * Trigger a delivery using the tool [trigger-pipeline](.circleci/trigger-pipeline).
-If no branch is specified, main is used as default. Per default the delivery is set to development
-* For more information on how to use it, execute it without parameters to see usage information.
+  * If no branch is specified, main is used as default. This should normally not be changed.
+  * Per default a development delivery is made.
+  * For more information on how to use it, execute it without parameters to see usage information.
 
 ## Workflows
 
 Several workflows exist for different purposes:
-* **commit**: Run for all commits of PRs to ensure good code quality and working code. Delivers Integreat, Malte and Aschaffenburg webapps on the main branch.
-* **scheduled_native_development_delivery**: Delivers Integreat, Malte and Aschaffenburg native builds to development (Testflight and Beta channel)
-twice a month.
-* **triggered_native_development_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers Integreat, Malte and Aschaffenburg builds for native to development.
-* **scheduled_production_delivery**: Delivers Integreat, Malte and Aschaffenburg native builds to production twice a month.
-* **triggered_production_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers Integreat, Malte and Aschaffenburg builds for native and web to production.
+* **commit**: Executed for all commits of PRs to ensure good code quality and working code. Delivers web development builds to https.//webnext.\<domain>.
```

Comment:
```suggestion
* **commit**: Executed for all commits of PRs to ensure good code quality and working code. Delivers web development builds to https://webnext.<domain>.
```

Reply:
Actually the backslash is needed to escape `<`.